pkg/kubelet/metrics/metrics.go

```diff
@@ -773,6 +775,27 @@ var (
 		},
 	)
 
+	// CPUManagerSharedPoolSizeMilliCores reports the current size of the shared CPU pool for non-guaranteed pods
+	CPUManagerSharedPoolSizeMilliCores = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           CPUManagerSharedPoolSizeMilliCoresKey,
+			Help:           "The size of the shared CPU pool for non-guaranteed QoS pods, millicores.",
```

Comment:
nit: `, millicores` doesn't read right grammatically, and it's not that much longer to write `in millicores`

Reply:
just me being sloppy in english ;) fixed, thanks

Reply:
since I only know one language, I might as well be annoying about it :)